### PR TITLE
Removed the react-native test.

### PR DESCRIPTION
**Pull request by excitement-engineer**
This PR removes the only react-native test in the repo. I don’t really see the value of including this test in the repo, in addition it adds complexity with the jest configurations and adds some other devDependencies which can now be removed.

### CODE REVIEW

**Comment by excitement-engineer** [comment]
Note, this is WIP. I am completing this as soon as I find some more time.

**Comment by excitement-engineer** [comment]
cc @rosskevin @jbaxleyiii

**Comment by rosskevin** [comment]
@excitement-engineer can you rebase - I'd like to get this one merged.

File: test/test-utils/shim.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/**
+ * React requires the requestAnimationFrame to be defined for tests.
+ */
+// Prevent typescript from giving error.
+const globalAny: any = global;
+
+globalAny.requestAnimationFrame = callback => {
+  setTimeout(callback, 0);
```
Comment (rosskevin) [inline test/test-utils/shim.ts]: This is the long way to type it, you could just do `(global as any).requestAnimationFrame`

**Comment by excitement-engineer** [comment]
It seems like the ci is not starting, maybe travis is down. The ci will not pass because of the broken test. Given that it seems to be testing something internal in apollo-client. Perhaps we should simply delete the test

**Comment by rosskevin** [comment]
If you resolve conflicts and push a change, it will run.

**Pull request merged**